### DISCUSSION TITLE
Is WinterCMS patched for OC Security advisor CVE-2021-32648?

### DISCUSSION THREAD

**Discussion by Tony-S** [wintercms/winter#406] [Q&A] [answered]
I've just received an October security advisory for CVE-2021-32648. Is WinterCMS affected by this vulnerability?
Thanks.
Tony

**Comment by LukeTowers** [comment, marked as answer]
Yes, we've been patched since v1.1.3 in April last year. We'll be releasing security advisories of our own shortly.

While the October team has proven unwilling to share security issues that could be harmful to our users as well as theirs; we do monitor their 1.0 and 1.1 branches for any changes that could indicate a risk for our users as well.

**Reply by Tony-S** [reply]
Thanks Luke! Great job.
Tony
